storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -1654,6 +1657,10 @@ private AssignmentDistributionService getAssignmentsDistributer() {
         return assignmentsDistributer;
     }
 
+    private StormMetricsRegistry getMetricsRegistry() {
```

Comment:
This seems redundant since it is private.  Any real reason for having this?

Reply:
It's used to access class member in static methods in the same way nimbus.getAssignmentsDistributer().

I'm fine with using the member directly but then should do the same with getAssignmentsDistributer()

Reply:
Thanks.  I missed that.